tasks/main.yml

```diff
@@ -1,11 +1,16 @@
 ---
+- name: Get mac user's name
+  shell: whoami
+  register: mac_user
```

Comment:
Just in case someone has a `mac` role that might have this variable, please change to `homebrew_whoami` (something more along those lines, so it won't blow out some other role's likely variable name).